lib/cli/commands/listpairs.ts

```diff
@@ -1,11 +1,38 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPairsRequest } from '../../proto/xudrpc_pb';
+import { ListPairsRequest, ListPairsResponse } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { getPairId } from '../../utils/utils';
+
+const HEADERS = [
+  colors.red('PairId'),
+  colors.green('Base'),
+  colors.red('Quote'),
```

Comment:
These colors in a pattern feels a little too "christmasy" to me. I know we're using red/green on the orders table but it makes more sense there where red is "sell" and green is "buy." Personally I'd stick to a single color for all the headers here, not sure which would look nicest, maybe blue which is also sort of the exchange union color.